### DISCUSSION TITLE
ios compatibility matrix

### DISCUSSION THREAD

**Discussion by andysousa** [ionic-team/capacitor#5297] [Q&A] [answered]
Anyone know where I might be able to find the ios compatibility matrix? We are trying to force users to a certain minimum ios version, but I am having a hard time finding the compatibility list for capacitor.  I am guessing there has to be certain capacitor apis which will trigger apple to force different ios compatibility...

Any thoughts on where to find this?

**Comment by jcesarmobile** [comment, marked as answer]
I'm not sure what do you mean by "compatibility matrix"

Capacitor 3 supports iOS 12+ 
https://capacitorjs.com/docs/ios#ios-support

Our official plugins also support iOS 12+, but some features might not be possible in all iOS versions, in example multiple image picking in Camera plugin only works on iOS 14+. Those kind of things are documented on each plugin function types and plugin docs page.

**Reply by andysousa** [reply]
I think that answers my question. Sorry if I was confusing. 

Some examples might be I know some new api changes are only in 13+, but sounds like Capacitor doesn't utilize those and I wanted my app to only work in 13+ I would need to find a plug-in that forces me to that version.  That sound about right?

**Reply by jcesarmobile** [reply]
The minimum iOS version supported by Capacitor is 12, but you can increase the minimum version your app supports if you want, can't be lower than 12 as you use Capacitor, but can be greater.

You can change the deployment target value from Xcode, and you should also change the value in the `ios/App/Podfile` file.
[This](https://developer.apple.com/library/archive/documentation/ToolsLanguages/Conceptual/Xcode_Overview/WorkingwithTargets.html) are Apple docs for that, the screenshoots don't look up to date, but they might help.
You can also change it project wide by clicking the project instead of the target and it should change the value for all targets in case you have more than 1.

**Reply by andysousa** [reply]
perfect, thanks for the clarification
